entrypoint.sh

```diff
@@ -19,7 +21,12 @@ fi
 
 attempt=1
 while [ $attempt -le "$INPUT_RETRIES" ]; do
-  sh -c "cf8 $INPUT_COMMAND"
+  if [ $INPUT_SKIP_SSL_VALIDATION -eq "true"]; then
+    sh -c "cf8 $INPUT_COMMAND --skip-ssl-validation"
+  else
+     sh -c "cf8 $INPUT_COMMAND"
+  fi
+
```

Comment:
use options flags for this behaviour 

```suggestion
  # on top of the script
  CF_CLI_OPTIONS=""
  
  if [ $INPUT_SKIP_SSL_VALIDATION -eq "true"]; then
    CF_CLI_OPTIONS="$CF_CLI_OPTIONS --skip-ssl-validation"
  fi
  
  # inside the while loop then
  # also replace "cf8" with "cf"
  sh -c "cf $INPUT_COMMAND $CF_CLI_OPTIONS"

```

Reply:
sorry my bad. cf8 is fine (dont replace with cf)